.travis.yml

```diff
@@ -30,6 +30,7 @@ matrix:
 before_install:
   - curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.0/install.sh | bash && . ~/.jabba/jabba.sh
   - if [ $SBT_LOCAL == true ]; then sbt -Dsbt.io.virtual=false publishLocalBin; fi
+  - rm -r $(find $HOME/.sbt -name "*-SNAPSHOT") || true
```

Comment:
> If the sbt snapshot builds are cached, then we don't actually dogfood
sbt when we publish a local sbt version because sbt will load with the
version from the last build

😱

Reply:
This should be `$HOME/.sbt/boot`.